python/pyspark/mllib/util.py

```diff
@@ -169,6 +169,22 @@ def loadLabeledPoints(sc, path, minPartitions=None):
         minPartitions = minPartitions or min(sc.defaultParallelism, 2)
         return callMLlibFunc("loadLabeledPoints", sc, path, minPartitions)
 
+    @staticmethod
+    def appendBias(data):
+        """
+        Returns a new vector with `1.0` (bias) appended to the input vector.
+        """
+        vec = _convert_to_vector(data)
```

Comment:
Will this work if "data" is a Vector type?  _convert_to_vector will leave it as a Vector type.